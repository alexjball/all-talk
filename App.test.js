import React from 'react'
import { render } from '@testing-library/react-native'
import App from './App'

it('renders', () => {
  const { baseElement, getByHintText } = render(<App />);

  expect(getByHintText('greeting')).toHaveTextContent(/hello/i);
  expect(baseElement).toMatchSnapshot();
});