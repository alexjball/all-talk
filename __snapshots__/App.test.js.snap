// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <Text
        accessibilityHint="greeting"
      >
        Hello there!
      </Text>
    </View>
  </View>
</View>
`;
